WordPress/Tests/WP/PostsPerPageUnitTest.inc

```diff
@@ -0,0 +1,30 @@
+<?php
+
+$args = array(
+	'posts_per_page' => 999, // Bad.
+	'posts_per_page' => -1, // OK.
```

Comment:
why is this OK? on VIP -1 was set as Bad

Reply:
found a ticket for this: https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/issues/1379

Reply:
That ticket is not the right one. See ticket #1157.

The original sniff has been split into two different sniffs:
* A WP one which checks for high pagination limits
* A VIP one which checks for disabling of pagination,

The VIP one will be removed in WPCS 2.0.

Reply:
thanks @jrfnl for the quick reply!

the WP doesn't check for -1, which is unlimited, in my opinion, worse than a high limit.
should it check also for it?